docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :bug:`-` Users will be able to see the address of the asset instead of the redundant asset name.
```

Comment:
Without specifying the location the changelog makes no sense

```suggestion
* :bug:`-` Users will be able to see the address of the asset instead of the redundant asset name in the newly detected tokens view.
```